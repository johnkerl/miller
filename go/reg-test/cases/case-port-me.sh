
# mention LHS value on first record should result in ZYX for process creation
export indir; run_mlr --from $indir/abixy put -q 'ENV["ZYX"]="CBA".NR; print | ENV["indir"]."/env-assign.sh" , "a is " . $a'
