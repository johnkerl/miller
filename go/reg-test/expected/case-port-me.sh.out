
mlr --from ./reg-test/input/abixy put -q ENV["ZYX"]="CBA".NR; print | ENV["indir"]."/env-assign.sh" , "a is " . $a
[ZYX]=[CBA1] a is pan
[ZYX]=[CBA1] a is eks
[ZYX]=[CBA1] a is wye
[ZYX]=[CBA1] a is eks
[ZYX]=[CBA1] a is wye
[ZYX]=[CBA1] a is zee
[ZYX]=[CBA1] a is eks
[ZYX]=[CBA1] a is zee
[ZYX]=[CBA1] a is hat
[ZYX]=[CBA1] a is pan

